<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_cruise_select</class>
 <widget class="QDialog" name="dlg_cruise_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reise auswählen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="Frm_Cruise" name="frm_cruise" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>248</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn_search">
     <property name="text">
      <string>Süchen</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btn_select">
     <property name="text">
      <string>Auswählen</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTableView" name="tv_cruises"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Frm_Cruise</class>
   <extends>QWidget</extends>
   <header>frm_cruise_code</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tv_cruises</tabstop>
  <tabstop>btn_search</tabstop>
  <tabstop>btn_select</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
